### ISSUE TITLE
Logging fix for SubmitBenefitsIntakeClaim job

### ISSUE THREAD

**Issue by micahaspyr** [department-of-veterans-affairs/va.gov-team#79431]
In the perform method of the SubmitBenefitsIntakeClaim job, the logging message sent to sentry  on a successful submission is incorrect. It should say 'Lighthouse::SubmitBenefitsIntakeClaim succeeded' instead of 'CentralMail::SubmitSavedClaimJob succeeded'.  All log messages also need to remove CentralMail:: and replace with Lighthouse::

<img width="850" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBemNYQkE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--b8449da1d196d977a9c7bc56774995d718301f89/Screenshot%202024-03-27%20at%2010.56.18%E2%80%AFAM.png" alt="Screenshot 2024-03-27 at 10.56.18 AM.png" />

**Comment by micahaspyr** [comment]
A needed reviewer was unable to get this merged today.  This NEEDS to be merged before a Burials changeover to make sure we can track the propagation of this job to the BI endpoint

**Comment by micahaspyr** [comment]
@tblackwe this was just merged into master!
